spec/omnicache/store_spec.rb

```diff
@@ -24,7 +24,13 @@ def self.load(value)
     end
   end
 
+  shared_context "without Datadog" do
+    before { hide_const("Datadog::Tracing") }
+  end
```

Comment:
If datadog is assumed to be available and present for all of these methods in prod (like I believe it is), then we probably want to run all tests without this `shared_context`